ci/test/00_setup_env_i686_centos.sh

```diff
@@ -7,6 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export HOST=i686-pc-linux-gnu
+export DOCKER_NAME=centos_7
```

Comment:
```suggestion
export DOCKER_NAME=ci_i686_centos_7
```

to avoid name collisions with other local containers named centos_7